spikeinterface/postprocessing/principal_component.py

```diff
@@ -359,7 +359,7 @@ def _fit_by_channel_local(self, sparsity, n_jobs, progress_bar):
                 sparse_channel_ids = sparsity[unit_id]
             else:
                 sparse_channel_ids = channel_ids
-            sparse_channel_inds = we.recording.ids_to_indices(sparse_channel_ids)
+            sparse_channel_inds = we.ids_to_indices(sparse_channel_ids)
```

Comment:
`ids_to_indices` should not be in waveformectractor it is too ampbiguous (channel or units???)

I think this ids__to_indicies should available only is the recording is here.

Reply:
We use it everywhere in the widgets..we could call it `channel_ids_to_indices`